config/roles.yml

```diff
@@ -272,25 +272,41 @@ cross_cluster_search_remote_full_access:
         - 'indices:data/read/search'
 
 # Allow users to operate query assistant
-ml_query_assistant_access:
+query_assistant_access:
```

Comment:
The rename of this item can create some confusion.  For new clusters they will have `query_assistant_access`, for older clusters, they will not get this new name/permissions.  So you can have clusters running 2.13 that have either `ml_query_assistant_access` or `query_assistant_access`, even with the new upgrade API [1] admins might have trouble knowing when to use one or the other.

I'd recommend keeping the old name, if the new name is important, duplicate the role with both names to avoid misalignment.  Changing from `ml_query_assistant_access` -> `query_assistant_access` is a manual change for admins.

- [1] https://github.com/opensearch-project/documentation-website/pull/6634

Reply:
Hi Peter, but this role has not been released yet, will that matter?